specutils/io/default_loaders/wcs_fits.py

```diff
@@ -139,14 +212,20 @@ def wcs1d_fits_writer(spectrum, file_name, hdu=0, update_header=False, **kwargs)
     spectrum : :class:`~specutils.Spectrum1D`
     file_name : str
         The path to the FITS file
-    hdu : int
+    hdu : int, optional
         Header Data Unit in FITS file to write to (base 0; default primary HDU)
-    update_header : bool
+    update_header : bool, optional
         Update FITS header with all compatible entries in `spectrum.meta`
-    unit : str or :class:`~astropy.units.Unit`
-        Unit for the flux (and associated uncertainty)
-    dtype : str or :class:`~numpy.dtype`
-        Floating point type for storing flux array
+    flux_name : str, optional
+        HDU name to store flux spectrum under (default 'FLUX')
+    mask_name : str or `None`, optional
+        HDU name to store mask under (default 'MASK'; `None`: do not save)
+    uncertainty_name : str or `None`, optional
+        HDU name to store uncertainty under (default set from type; `None`: do not save)
+    unit : str or :class:`~astropy.units.Unit`, optional
+        Unit for the flux (and associated uncertainty; defaults to `spectrum.flux.unit`)
```

Comment:
Unless `spectrum.flux.unit` actually generates API doc, use double-backticks. I cannot check because RTD build failed on your PR. Also see #1028.

Reply:
[`spectrum.flux`](https://specutils.readthedocs.io/en/stable/api/specutils.Spectrum1D.html#specutils.Spectrum1D.flux) does, but not specifically `unit`, is that what you mean? I don't know if the loaders themselves generate API docs, or rather where they might be linked from ;-)
Thanks for pointing out the RTD failure; throws in all current PRs and does not seem related – from a quick glance some parsing error related to `reproject`, but needs further investigation.

Reply:
Actually it seems none of the functions in `default_loaders` have any docs built at all, though it would certainly be useful to have them and link them in https://specutils.readthedocs.io/en/latest/spectrum1d.html#list-of-loaders.